gulpfile.mjs

```diff
@@ -0,0 +1,67 @@
+import {
+  buildCSS,
+  buildJS,
+  generateManifest,
+  runTests,
+  watchJS,
+} from '@hypothesis/frontend-build';
+import gulp from 'gulp';
+
+// import tailwindConfig from './tailwind.config.mjs';
```

Comment:
The CSS and testing parts of this are not used yet. This will come in follow-up PRs.